app/assets/v2/js/pages/bounty_details.js

```diff
@@ -164,13 +164,26 @@ var callbacks = {
 
     expires_date = new Date(val);
     now = new Date(result['now']);
-    var response = timeDifference(now, expires_date);
 
-    if (new Date(val) < new Date()) {
+    var percent = 100 * (
```

Comment:
Please forgive the nitpick, but would something like percentCompleted be a bit more explicit? 😛

Reply:
@mkosowsk kept it as `expiringInPercentage`. 
fixing this before @mbeacom sees this and judges me :P

Reply:
:)

Reply:
🤷‍♂️  No judgement here!